### PR TITLE
Mention 'dropped by the kernel'

### PR DESCRIPTION
**Pull request by jotak**
image

image

### CODE REVIEW

File: web/src/utils/overview-panels.ts [hunk 1]
```diff
@@ -99,19 +99,19 @@ export const getOverviewPanelInfo = (
...
         chartType: t('donut'),
-        tooltip: t('The top drop causes over the selected interval')
+        tooltip: t('The top drop causes (dropped by the kernel) over the selected interval')
       };
     case 'top_dropped_bar_total':
       return {
         title: t('Top {{limit}} {{type}} dropped rates stacked with total', { limit, type }),
         chartType: t('bars'),
-        tooltip: t('The top dropped rates as bar compared to total as line over the selected interval')
+        tooltip: t('The top dropped rates (dropped by the kernel) as bar compared to total as line over the selected interval')
```
Comment (msherif1234) [inline web/src/utils/overview-panels.ts:114]: probably here too ?

Comment (jotak) [reply web/src/utils/overview-panels.ts]: it was a formatting issue .. running `make fmt` fixed it, PR updated

Comment (msherif1234) [reply web/src/utils/overview-panels.ts]: /lgtm

**Pull request merged**